lib/palindrome_check.rb

```diff
@@ -1,5 +1,25 @@
-# A method to check if the input string is a palindrome.
-# Return true if the string is a palindrome. Return false otherwise.
 def palindrome_check(my_phrase)
-  raise NotImplementedError
+  if my_phrase.nil?
+    return false
+  end
+
+  start_counter = 0
+  end_counter = my_phrase.length - 1
+
+  until start_counter >= end_counter
+    while my_phrase[start_counter] == " "
```

Comment:
Also check that start_counter continues to be >= end_counter.
test case: an input with just white spaces.